builtin/providers/google/resource_compute_instance_group_manager.go

```diff
@@ -165,6 +207,22 @@ func resourceComputeInstanceGroupManagerRead(d *schema.ResourceData, meta interf
 	d.Set("target_size", manager.TargetSize)
 	d.Set("self_link", manager.SelfLink)
 
+	// Prepare the list of named ports
+	namedPortsCount := d.Get("named_port.#").(int)
+
+	namedPorts := make([]map[string]interface{}, 0, 1)
```

Comment:
if these are returned in a different order than the schema stored them in, will `reflect.DeepEqual()` as used in `helper/schema/resource_data.go` see them as equal?

e.g. 
`{0: {"name": "http", "port": 8080}, 1: {"name": "https", "port" 8443}}`
`{0: {"name": "https", "port": 8443}, 1: {"name": "http", "port" 8080}}`